src/main/resources/templates/mail/notification/sshKeyAddedEmail.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+
+<th:block th:replace="mail/notification/fragments :: head"/>
+
+<body>
+<!-- Message Body Start -->
+<th:block th:replace="mail/notification/fragments :: header" />
```

Comment:
I recently applied changes in #10114 to avoid deprecations. Please adapt all email templates accordingly

Reply:
I adapted the emails :+1: